.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,82 @@
+name: Bug report
+
+# See the json schema to help you create / update this form
+# https://json.schemastore.org/github-issue-forms.json
+
+description: Fill in this template to report a bug
+
+title: "[BUG] "
+
+labels: ["type:bug"]
```

For now the repo only has the `bug` label (not `type:bug`)

```suggestion
labels: ["bug"]
```

---

Modified to the new labels we created